lib/puppet_webhook.rb

```diff
@@ -9,10 +9,15 @@ class PuppetWebhook < Sinatra::Base # rubocop:disable Style/Documentation
       parsers: { 'application/json' => Sinatra::Parsers::WebhookJsonParser.new },
       handlers: { 'application/json' => proc { |e, type| [400, { 'Content-Type' => type }, [{ error: e.to_s }.to_json]] } }
   register Sinatra::ConfigFile
-  config_file '../config.yml'
+  if File.exist?('/etc/puppet-webhook/app.yml')
```

Comment:
Should we have a ```CONFIG_DIR``` setting somewhere?

Reply:
Would it provide us with anything useful?

Seems like it would just be another config option just to support the config files. Seems a bit redundant. Unless you mean to set it to one of [ static value, CLI passed arg ]

Reply:
Running from a git checkout might be easier, or as non-root user. FreeBSD has ```/usr/local/etc``` instead. There are some odd setups :)